cli/tink/cmd/template/create.go

```diff
@@ -62,7 +62,7 @@ func readTemplateData() []byte {
 }
 
 func createTemplate(c *cobra.Command, args []string) {
-	req := template.WorkflowTemplate{Name: templateName, Data: readTemplateData()}
+	req := template.WorkflowTemplate{Name: templateName, Data: string(readTemplateData())}
```

Comment:
Since the returned value of `readTemplateData` is converted to `string` in most of the places, I think it would make sense to change the return type to `string` from `[]byte`.